src/main.rs

```
@@ -0,0 +1,11 @@
+fn main() {
+    let x: Vec<u8> = vec![];
+    let y = x.len() == 0;
+
+    let mut a = 5;
+    a = a;
```

clippy

value assigned to `a` is never read

(warning)